<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2007, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

  </comment>
 <exportmacro></exportmacro>
 <class>qtractorPluginForm</class>
 <widget class="QWidget" name="qtractorPluginForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>48</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>1</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font" >
   <font>
    <family>Sans Serif</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>qtractorPluginForm</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/qtractor.qrc" >:/icons/qtractorTracks.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="PresetComboBox" >
       <property name="minimumSize" >
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Preset name</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="SavePresetToolButton" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Save preset</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../icons/qtractor.qrc" >:/icons/formSave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DeletePresetToolButton" >
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Delete preset</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../icons/qtractor.qrc" >:/icons/formRemove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="ActivateToolButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Activate plugin</string>
       </property>
       <property name="text" >
        <string>Active</string>
       </property>
       <property name="icon" >
        <iconset resource="../icons/qtractor.qrc" >:/icons/itemLedOff.png</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>PresetComboBox</tabstop>
  <tabstop>SavePresetToolButton</tabstop>
  <tabstop>DeletePresetToolButton</tabstop>
  <tabstop>ActivateToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/qtractor.qrc" />
 </resources>
 <connections/>
</ui>
